### ISSUE TITLE
gazelle ignores vendor inside vendor dependency

### ISSUE THREAD

**Issue by excavador** [bazelbuild/bazel-gazelle#91]
Copy of https://github.com/bazelbuild/rules_go/issues/1245

Verified on latest bazel (0.9.0) and latest rules_go (commit hash 93fbdfae183f24c1bea4dc6f776188523ba3a2be)

Please take a look to repo
https://github.com/excavador/bazel_vendor_inside_vendor

**Comment by jayconrod** [comment]
Thanks for reporting. I reproduced this with 0.8, but not at master. I believe this was fixed recently, probably in #89. I'm planning to tag a release this week which will include that. I'll close this issue now.

You'll probably want to use the [new setup instructions](https://github.com/bazelbuild/bazel-gazelle#running-gazelle-with-bazel). The old `gazelle` rule in rules_go will be removed at some point.
